OpenRA.Mods.Common/Widgets/VideoPlayerWidget.cs

```diff
@@ -46,7 +46,14 @@ public void Load(string filename)
 			if (filename == cachedVideoFileName)
 				return;
 
-			var video = VideoLoader.GetVideo(Game.ModData.DefaultFileSystem.Open(filename), true, Game.ModData.VideoLoaders);
+			var stream = Game.ModData.DefaultFileSystem.Open(filename);
+			if (stream.Length == 0)
+			{
+				Log.Write("debug", $"Empty dummy file {filename} detected.");
+				return;
+			}
+
+			var video = VideoLoader.GetVideo(stream, true, Game.ModData.VideoLoaders);
 			Open(video);
```

Comment:
While we're here, can you put a null check inside of `Open`? `GetVideo` might return null.

Reply:
Added.